suffuse/path_test.go

```diff
@@ -1,9 +1,19 @@
 package suffuse
 
 import (
+  "time"
   . "gopkg.in/check.v1"
 )
 
+func NewDuration(spec string) time.Duration {
+  dur, err := time.ParseDuration(spec)
+  MaybePanic(err)
+  return dur
+}
+func Hours(n int) time.Duration {
+  return NewDuration(Sprintf("%d", n) + "s")
+}
+
 func (s *Tsfs) TestPathOsMethods(c *C) {
```

Comment:
I think this method should be split up into methods with more descriptive names.